src/python/grpcio/grpc/_channel.py

```diff
@@ -142,7 +142,7 @@ def _handle_event(event, state, response_deserializer):
         response = _common.deserialize(
             serialized_response, response_deserializer)
         if response is None:
-          details = b'Exception deserializing response!'
+          details = 'Exception deserializing response!'
```

Comment:
This sounds like a mistake on my part - details strings should always be bytes, from the C Core to the application, and in both Python 2 and 3. I apologize that I didn't something with `format` that interfered with that.